pd/src/main.rs

```diff
@@ -153,66 +177,285 @@ async fn main() -> anyhow::Result<()> {
                 x = thin_wallet_server => x?.map_err(|e| anyhow::anyhow!(e))?,
             };
         }
-        Command::CreateGenesisTemplate => {
-            use penumbra_crypto::keys::SpendKey;
+        Command::GenerateTestnet {
+            num_validator_nodes,
+            // TODO this config is gated on a "populate persistent peers"
+            // setting in the Go tendermint binary. Populating the persistent
+            // peers will be useful in local setups until peer discovery via a seed
+            // works.
+            starting_ip,
```

Comment:
is the unusedness of this variable related to the TODO comment?

Reply:
Yes. It will be fixed by #228